bentoml/service_env.py

```diff
@@ -82,6 +82,17 @@ def write_to_yaml_file(self, filepath):
         with open(filepath, 'wb') as output_yaml:
             self._yaml.dump(self._conda_env, output_yaml)
 
+    def conda_env_yaml(self, filepath):
```

Comment:
simply do `self._conda_env = self._yaml.load(filepath)` here and then 1) verify the python version matches current python session's python versions, 2) modify the env name as it is used for activating the env in docker image

Reply:
okay 👍

Reply:
Actually, that approach may lead to losing other dependencies specified in `self._conda_env`. If you could merge two `yaml` that'd be ideal, you may refer to the official conda-merge's code here; https://github.com/amitbeka/conda-merge/blob/master/conda_merge.py

Reply:
Oh, I didn't realize that. Thanks for pointing it out :). 
I'll update the draft for your review again.

Reply:
Does doing something like this will serve the purpose?

` def conda_env_yaml(self, filepath):
        dep=set()
        conda_yaml = self._yaml.load(filepath)
        #check python version
        if self._conda_env['python_version']==conda_yaml['python_version']:

                #merge conda_yaml and _conda_env
                dep.add(self._conda_env['dependencies'])
                dep.add(conda_yaml['dependencies'])
                conda_yaml['dependencies']=dep
        else:
            raise "some error"`

Reply:
I think support merging two conda environment file is just way too error-prone and hard to implement.

Could we just add a class method for initializing a CondaEnv instance with an env.yml file? that should be a lot easier to implement.

Reply:
Hi, apologies for being away, I was caught up with university applications.
After adding this parameter we would take the config_file path as an input and then? How are we going to merge the dependencies? 

 ```
 def write_to_yaml_file(self, filepath):
        with open(filepath, 'wb') as output_yaml:
            self._yaml.dump(self._conda_env, output_yaml)
        if self.env_config_file:
            config_file_env=self._yaml.load(self.env_config_file)
            self._yaml['dependencies'].append(config_file_env['dependencies'])
            print(_yaml['dependencies'])
```

As you mentioned earlier I also think we might have to check the python version before doing this ^^